### ISSUE TITLE
lstm_generator_shakespeare.py: Warnings for 'list' object has no attribute 'name'

### ISSUE THREAD

**Issue by avantol** [tflearn/tflearn#631]
Yes, I see this is only a warning... but newbies like me will wonder what to do about warnings on their first attempt at running tensorflow.

Safely ignore? Fix?

(BTW: Thanks for making Tensorflow possible!)

========================
Loading previous char_idx
Vectorizing text...
Text total length: 4,573,338
Distinct chars   : 67
Total sequences  : 1,524,438
---------------------------------
Run id: shakespeare
Log directory: /tmp/tflearn_logs/
WARNING:tensorflow:Error encountered when serializing layer_tensor/LSTM_1.
Type is unsupported, or the types of the items don't match field type in CollectionDef.
'list' object has no attribute 'name'
WARNING:tensorflow:Error encountered when serializing layer_tensor/LSTM.
Type is unsupported, or the types of the items don't match field type in CollectionDef.
'list' object has no attribute 'name'
WARNING:tensorflow:Error encountered when serializing layer_tensor/Dropout.
Type is unsupported, or the types of the items don't match field type in CollectionDef.
'list' object has no attribute 'name'
---------------------------------
Training samples: 1371994
Validation samples: 152444
...etc.
================================

### What related GitHub issues or StackOverflow threads have you found by searching the web for your problem?

None

### Environment info
Operating System:
Linux Mint 17.3
#64-Ubuntu SMP Mon Sep 22 21:28:38 UTC 2014
3.13.0-37-generic

Installed version of CUDA and cuDNN: 
(please attach the output of `ls -l /path/to/cuda/lib/libcud*`):

n/a

If installed from binary pip package, provide:

1. A link to the pip package you installed:
2. The output from `python -c "import tensorflow; print(tensorflow.__version__)"`.

n/a

If installed from source, provide 

1. The commit hash (`git rev-parse HEAD`)
be8a271f00be42aa61c24d712fc2ff593ac49345

2. The output of `bazel version`
Build label: 0.4.4
Build target: bazel-out/local-fastbuild/bin/src/main/java/com/google/devtools/build/lib/bazel/BazelServer_deploy.jar
Build time: Wed Feb 1 18:54:21 2017 (1485975261)
Build timestamp: 1485975261
Build timestamp as int: 1485975261

### If possible, provide a minimal reproducible example (We usually don't have time to read hundreds of lines of your code)

Simply run the code (unmodified) at
https://github.com/tflearn/tflearn/blob/master/examples/nlp/lstm_generator_shakespeare.py

### What other attempted solutions have you tried?
None

### Logs or other output that would be helpful
(If logs are large, please upload as attachment or provide link).

See above
